### ISSUE TITLE
Add labels to ComplexSubjects

### ISSUE THREAD

**Issue by fsteeg** [hbz/lobid-resources#639]
For implementing efficient topic search in NWBib based on API 2.0, we need full subject chains in a single field, which can then be used in an aggregation. @dr0i suggested to add `label` fields to subjects with type `ComplexSubject`. So e.g. for https://lobid.org/resources/HT019305260.json we'd get these labels:

```json
{
    "componentList" : [ ... ],
    "type" : [ "ComplexSubject" ],
    "label" : "Rheinland | Geologie"
},{
    "componentList" : [ ... ],
    "type" : [ "ComplexSubject" ],
    "label" : "Köln-Klettenberg | Gebäude | Garten | Stein"
}
```

Additionally, we need a `raw` `keyword` subfield for the `subject.label` field in `index-config.json`:

```json
"label": {
    "type": "text",
    "fields": {
        "raw": {
            "type": "keyword",
            "index": "true"
        }
    }
}
```

The above is exactly what we already have for `subject.componentList.label` in the `572-updateEsTo5.6.3` branch. This could be copied for `subject.label`.

**Comment by fsteeg** [comment]
I realized that with the approach above, we'd still have the problem that we'd have to distinguish `subject.label`s in `ComplexSubject`s from the other `subject.label`s, so we can't simply create an aggregation for `subject.label`. Same for searching in that field when clicking a topic in the NWBib topic search. A straightforward fix for both the aggregation and the search issue would be to give the field a specific name, e.g. `subject.complexLabel`, `subject.summary`, or `subject.topic`.

On the other hand, there's probably a way make this work with all the options offered by Elasticsearch. @acka47 pointed me to the bucket selector aggregation, which sounds like it might work, see https://www.elastic.co/guide/en/elasticsearch/reference/5.6/search-aggregations-pipeline-bucket-selector-aggregation.html

**Comment by dr0i** [comment]
Deployed to production, closing.

**Comment by dr0i** [comment]
Ups, sorry, reopened again. I think this was unintentional be closed by #667, yes @fsteeg?

**Comment by fsteeg** [comment]
@dr0i No, it was fine to close, I reviewed the PR, and topic search is under review in https://github.com/hbz/nwbib/issues/163.
